src/Microsoft.DotNet.ProjectModel/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-rc4-*",
+  "version": "1.0.0-featmsbuild-*",
```

Comment:
it's strange that this nupkg was on an rc4 version. it seems delibirately different from others. Do you know why?